### ISSUE TITLE
Please build eu-central (Frankfurt) AMI IDs again

### ISSUE THREAD

**Issue by robyduck** [fedora-infra/fedimg#36]
Since a few releases we don't have the eu-central AMI ID anymore. It would be nice to have it again. I files also a releng ticket for it, just linking it here too.
https://fedorahosted.org/rel-eng/ticket/6259

**Comment by ralphbean** [comment]
@kushaldas, fedimg uses [this file](https://infrastructure.fedoraproject.org/cgit/ansible.git/tree/roles/fedimg/templates/fedimg.cfg#n35) for its list of regions in production.  I don't know why eu-central-1 isn't on there.  Perhaps it was just forgotten, or perhaps it was left out for some reason.

**Comment by ralphbean** [comment]
This is working now after #40.
